inc/classes/subscriber/preload/class-preload-subscriber.php

```diff
@@ -40,10 +51,12 @@ class Preload_Subscriber implements Subscriber_Interface {
 	 * @author Remy Perona
 	 *
 	 * @param Homepage     $homepage_preloader Homepage Preload instance.
+	 * @param Sitemap      $sitemap_preloader  Sitemap Preload instance.
 	 * @param Options_Data $options            WP Rocket Options instance.
 	 */
-	public function __construct( Homepage $homepage_preloader, Options_Data $options ) {
+	public function __construct( Homepage $homepage_preloader, Sitemap $sitemap_preloader, Options_Data $options ) {
 		$this->homepage_preloader = $homepage_preloader;
+		$this->sitemap_preloader  = $sitemap_preloader;
```

Comment:
Let's remove these changes. We don't need the sitemap preloader in this subscriber.